src/battle_tower.c

```diff
@@ -1051,24 +1052,24 @@ struct
 
 #include "data/battle_frontier/battle_tent.h"
 
-static void (* const gUnknown_085DF96C[])(void) =
-{
-    sub_8161F94,
-    sub_8162054,
-    sub_81620F4,
-    ChooseNextBattleTowerTrainer,
-    sub_81621C0,
-    AwardBattleTowerRibbons,
-    SaveBattleTowerProgress,
-    sub_8163914,
-    nullsub_61,
-    SpriteCB_Null6,
-    sub_81642A0,
-    sub_8164828,
-    sub_8164B74,
-    sub_8164DCC,
-    sub_8164DE4,
-    sub_8164E04,
+static void (* const sBattleTowerFuncs[])(void) =
+{
+    [BATTLE_TOWER_FUNC_0] = sub_8161F94,
+    [BATTLE_TOWER_FUNC_1] = sub_8162054,
+    [BATTLE_TOWER_FUNC_2] = sub_81620F4,
+    [BATTLE_TOWER_FUNC_CHOOSE_TRAINER] = ChooseNextBattleTowerTrainer,
+    [BATTLE_TOWER_FUNC_4] = sub_81621C0,
+    [BATTLE_TOWER_FUNC_GIVE_RIBBONS] = AwardBattleTowerRibbons,
+    [BATTLE_TOWER_FUNC_SAVE] = SaveBattleTowerProgress,
+    [BATTLE_TOWER_FUNC_7] = sub_8163914,
+    [BATTLE_TOWER_FUNC_NOP] = nullsub_61,
+    [BATTLE_TOWER_FUNC_NOP_SPRITE] = SpriteCB_Null6,
```

Comment:
This function is not related to a sprite or sprite callback.

Reply:
Seems these were introduced by #553, probably worth removing the others that also aren't sprite callbacks. In any case this ones fixed